rsts/community/troubleshoot.rst

```diff
@@ -105,52 +105,54 @@ Troubles with ``make start`` command in flytesnacks
+- FlyteCTL uses GRPC APIs of the FlyteAdmin to administer Flyte resources and in case of proxy settings, it uses an additional CONNECT handshake at GRPC layer to perfrom the same. Additonal info is available `here <https://github.com/grpc/grpc-go/blob/master/Documentation/proxy.md>`__
 
-- On windows environment it has been noticed that NO_PROXY variable doesn't work for bypassing the proxy settings. GRPC issue reported `here <https://github.com/grpc/grpc/issues/9989>`__ provides additional details though it doesn't seem to have been tested on windows yet.Inorder to get around this issue unset both the HTTP_PROXY and HTTPS_PROXY variables.
+- On Windows environment it has been noticed that ``NO_PROXY`` variable doesn't work to bypass the proxy settings. `This <https://github.com/grpc/grpc/issues/9989>`__ GRPC issue provides additional details though it doesn't seem to have been tested on Windows yet. Inorder to get around this issue, unset both the ``HTTP_PROXY`` and ``HTTPS_PROXY`` variables.
 
-Troubles with flytectl command with cloudfare DNS
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Troubles with ``flytectl`` command with Cloudfare DNS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- flytectl throws permission error with cloudfare DNS endpoint
-- Cloudflare instance by default would proxy the requests and would filter out grpc.
-- You have two options either enable grpc in the network tab or turn off the proxy.
+- FlyteCTL throws permission error with Cloudfare DNS endpoint
+- Cloudflare instance by default would proxy the requests and would filter out GRPC.
+- You have two options: 
+    - Either enable grpc in the network tab; OR
+    - Turn off the proxy.
 
-Troubles with flytectl command with auth enabled
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Troubles with ``flytectl`` command with auth enabled
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- flytectl uses OpenID connect if auth is enabled in the flyte environment.
-- It opens an http server port on localhost:53593 and has a callback endpoint for the OpendID connect server to call into for the response.
-- If the callback server call fails, then please check if flytectl failed in running the server.
-- Verify if you have an entry for localhost in your /etc/hosts file
-- It could also mean that the call back took longer and flytectl deadline expired on the wait which defaults to 15 secs.
+- The ``flytectl`` command uses OpenID connect if auth is enabled in the Flyte environment.
+- It opens an ``HTTP`` server port on localhost:53593. It has a callback endpoint for the OpendID connect server to call into for the response.
+- If the callback server call fails, please check if ``flytectl`` failed to run the server.
+- Verify if you have an entry for localhost in your ``/etc/hosts`` file.
+- It could also mean that the call back took longer and FlyteCTL deadline expired on the wait which defaults to 15 secs.
```

Comment:
```suggestion
- FlyteCTL commands use OpenID connect if auth is enabled in the Flyte environment
- It opens an ``HTTP`` server port on localhost:53593. It has a callback endpoint for the OpenID connect server to call into for the response
    - If the callback server call fails, please check if FlyteCTL failed to run the server
    - Verify if you have an entry for localhost in your ``/etc/hosts`` file
    - It could also mean that the callback took longer and the FlyteCTL deadline expired on the wait which defaults to 15 secs
```